server.ts

```diff
@@ -1,3 +1,5 @@
+process.env = Deno.env.toObject();
```

Comment:
```suggestion
// process.env = Deno.env.toObject();
import "data:application/javascript;charset=utf-8;base64,cHJvY2Vzcy5lbnYgPSBEZW5vLmVudi50b09iamVjdCgpOw=="
```
Otherwise the imports will be evaluated before your code to assing to process